cmd/housekeeper.go

```diff
@@ -13,6 +14,14 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	hkDefaultPprofEnabled    = os.Getenv("PPROF") == "1"
+	hkDefaultPprofListenAddr = common.GetEnv("PPROF_LISTEN_ADDR", "localhost:8080")
```

Comment:
hmm isn't port 8080 quite commonly used? might see port conflicts

Reply:
good point, updated to 9064 (9060 used by API, 9062 by website)

Reply:
good point, updated to 9064 (9060 used by API, 9062 by website)